src/core/search.js

```diff
@@ -16,7 +16,10 @@ let corefn = ({
       return eles.collection();
 
     } else if( is.array( eles ) ){
-      return new Collection( this, eles, opts );
+      if (!opts) {
+        opts = {};
+      }
+      return new Collection( this, eles, opts.unique, opts.removed );
```

Comment:
What do you think about setting `opts.removed` to `true` by default?

I.e.

- `cy.add(eleJsons)` is for creating elements that are now added
- `cy.collection(eleJsons)` is for creating elements that aren't yet added

> It seems like a pretty obscure use case, so I am fine not adding a separate API call for it. It is currently setup in this way so it works as it normally has unless they explicitly include the removed flag as true.

Reply:
Oh that's, what you mean, sure that would be good with me, and we could document that.

Reply:
@maxkfranz it seems collection is used as the main method throughout Cytoscape to add elements to the graph. Changing that to true, and thus skipping the restore causes 300+ tests to fail as well as cy.add() and the core initialization element list no longer adds elements to the graph.

Reply:
I thinks it's currently ready to go with option 2 I just need to add the documentation.

Reply:
Let me know when that's up, and we can merge this in for release